go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/Venafi/terraform-provider-venafi
 go 1.21
 
 require (
-	github.com/Venafi/vcert/v5 v5.6.5-0.20240516011016-3f129abb0388
+	github.com/Venafi/vcert/v5 v5.7.0-rc2.0.20240525040228-9a6c6b02a545
```

Comment:
could you update here to use `v5.7.0`?

Reply:
will do

Reply:
unfortunately, 5.7.0 does not contain the structs required for terraform to work. I will wait until 5.7.1 is released to change it